query/src/main/java/org/infinispan/query/impl/massindex/MassIndexerProgressMonitor.java

```diff
@@ -48,6 +48,14 @@ public void documentsAdded(long increment) {
       }
    }
 
+   public void preIndexingReloading() {
+      log.preIndexingReloading();
+   }
+
+   public void indexingStarting() {
+      log.indexingStarting();
+   }
+
    public void indexingCompleted() {
       log.indexingEntitiesCompleted(documentsDoneCounter.get());
```

Comment:
The Jira mentions that it should log the duration for the indexing

Reply:
Nice question. I didn't added it, since from what I can see we already log the progress of the reindexing operation every 50 elements: https://github.com/fax4ever/infinispan/blob/860892ba15967949b9be15557303a2c549ba2a45/query/src/main/java/org/infinispan/query/impl/massindex/MassIndexerProgressMonitor.java#L47

Reply:
As the progress message 14015 is only shown if you have %50 entries I would leave that to debug and show the message 14014 "reindexed X entities in X seconds" as a final summary

Reply:
@wfink you're right! I'm pushing the fix

Reply:
pushed

Reply:
Great.
The 14015 is still INFO, I think this is not necessary for INFO if a cache has huge amount of entries.
But maybe DEBUG will be too much logging and a customer might happy see that progress information.
All 50 sounds frequent to me, I hope 50 entries are indexed in seconds :), so maybe 100 or 1000?
In general fine with the change

Reply:
I agree. Made the changes here. Thanks for the good ideas!